src/main/resources/i18n/Enums_es.properties

```diff
@@ -82,6 +82,9 @@ public.Role.Contributor=Colaborador
 public.Role.Manager=Gerente
 public.Role.Owner=Propietario
 public.SeedStorageBehavior.Intermediate=Intermediario
+public.SeedStorageBehavior.LikelyIntermediate=Probablemente intermedio
+public.SeedStorageBehavior.LikelyOrthodox=Probablemente ortodoxo
+public.SeedStorageBehavior.LikelyRecalcitrant=Probable recalcitrante
```

Comment:
@constanzauanini Should this be "Probablemente" like the other two?

Reply:
Yes, "probablemente" would be the correct way to say it